test/common/link-patterns.json

```diff
@@ -0,0 +1,24 @@
+[
+    {
+        "label": "screenshot",
+        "pattern": "Wrote screenshots to ([A-Za-z0-9\\-\\.]+\\.png)$",
```

Comment:
This must be "Wrote screenshot to"

Reply:
This is not just cosmetical, right? It must match what testlib.py's snapshot() does, to actually recognize "normal" failure screenshots.

Reply:
Correct.

Reply:
Fixed.